### PR TITLE
docs: add full working example tutorial in misc/deployment/ with link in intro

### PR DESCRIPTION
**Pull request by rela-v**
Resolves https://github.com/itzg/docker-minecraft-server/issues/2222

### CODE REVIEW

File: docs/misc/deployment/docker-compose-full-example.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## 4. Create a New Server Directory
+
+Create a new folder and name it something recognizable (such as "minecraft-server", for example). You may either use the command line, or your system's file manager GUI.
+
+## 5. Create and Populate docker-compose.yml File in the Server Directory
+
+The most important part of the docker-compose.yml creation step is to make sure that your docker-compose.yml file is properly formatted. One potential example docker-compose.yml file is as follows (be sure to delete any irrelevant lines to your application, or add any that you may need before copying and pasting the file contents). Be sure that the indentation is correct for every part of the file. 
+
+!!!note
```
Comment (rela-v) [inline docs/misc/deployment/docker-compose-full-example.md:27]: There's a mistake here, missing a space! Didn't even notice it while I was pushing. Just letting you know that it is there.

Thank you once again for your attention towards this matter and for considering allowing me to contribute to this wonderful project.

**Comment by rela-v** [comment]
Should be good now, let me know if there's any other changes that should be made prior to merge. Thank you so much again.

- Rela

**Review by itzg — COMMENTED** [review]
This is great content!

It's such great content that it seems more like a technical article. I think I would prefer to see you publish this as your own article at Medium, dev.to, etc (I prefer dev.to these days). With that, it would be great to include a link to the article (and others) to this deployment section.

**Comment by rela-v** [comment]
> This is great content!
> 
> It's such great content that it seems more like a technical article. I think I would prefer to see you publish this as your own article at Medium, dev.to, etc (I prefer dev.to these days). With that, it would be great to include a link to the article (and others) to this deployment section.

Would you prefer a link in the intro, or to keep this section and have a short blurb wrapping the link?

**Comment by rela-v** [comment]
All suggested changes were added: awaiting further guidance on the placement of the link.

Thank you so much for the amount of time and effort you put into giving feedback in this code review. I sincerely appreciate it and I am certain that this can help many people who would like to use your project!!

- Rela

**Comment by itzg** [comment]
Let's go with a combination of the ideas:

Under https://github.com/itzg/docker-minecraft-server/tree/master/docs/misc/deployment add an `index.md` that includes for now just a section for "supporting articles". Within that section I really the idea of not only the article link, but the short blurb around it. That can be a pattern for any other articles to link from here. 

Thanks!

**Comment by rela-v** [comment]
> Let's go with a combination of the ideas:
> 
> Under https://github.com/itzg/docker-minecraft-server/tree/master/docs/misc/deployment add an `index.md` that includes for now just a section for "supporting articles". Within that section I really the idea of not only the article link, but the short blurb around it. That can be a pattern for any other articles to link from here.
> 
> Thanks!

Definitely - I've added the blurb in the new index.md file per your recommendation, thank you so much for your time and for being so receptive!

- Rela

**Review by itzg — APPROVED** [review]
Very good!

**Pull request merged**